.github/test-build

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# Remember where we started; this is where our source checkout lives.
+src=`pwd`
+
+# Load the module command and use it to load MPI.
+. /etc/profile.d/modules.sh
+module load mpi
+
+# Configure sccache to use a docker volume
+export SCCACHE_DIR=/sccache
+export SCCACHE_CACHE_SIZE="1G"
+echo "::group::Setup"
+echo "Cache size: "
+du -sh /sccache
+sccache -s
+echo "::endgroup::"
+
+# Create a directory to hold the build, then build and test IBAMR
+mkdir /bld
+cd /bld
+echo "::group::Configure"
+cmake \
+  -DCMAKE_C_COMPILER_LAUNCHER:STRING=sccache \
+  -DCMAKE_CXX_COMPILER_LAUNCHER:STRING=sccache \
+  -DCMAKE_Fortran_COMPILER_LAUNCHER:STRING=sccache \
+  -DCMAKE_C_FLAGS="-O3 -fPIC -march=native" \
+  -DCMAKE_CXX_FLAGS="-O3 -fPIC -march=native" \
+  -DCMAKE_Fortran_FLAGS="-O3 -fPIC -march=native" \
```

Comment:
Why was adding `-fPIC` necessary? The default for CMake is to build with shared libraries (so this flag should always be on by default).

Reply:
It may not be... I have pushed a new version without it (and the build is running now). It was unclear whether SAMRAI or IBAMR needed the flag, so I added it to both.

Reply:
SAMRAI definitely needs it (and thats in the docker images) but we should not.